mangaki/mangaki/management/commands/dedupe.py

```diff
@@ -0,0 +1,216 @@
+from django.core.management.base import BaseCommand, CommandError
+
+import os
+import csv
+from dedupe import Dedupe, consoleLabel, canonicalize
+from mangaki.models import Work
+
+
+
+
+class Command(BaseCommand):
+    args = ''
+    help = 'Make training of dedupe with mangas or animes'
+
+    def add_arguments(self, parser):
+        parser.add_argument('myargs', nargs='+', type=str)
```

Comment:
Let's make it a `category` single argument rather than an array arguments, otherwise, the automatically generated help won't make a lot of sense.